flink-python/pyflink/table/statement_set.py

```diff
@@ -50,6 +50,13 @@ def add_insert_sql(self, stmt: str) -> 'StatementSet':
         """
         self._j_statement_set.addInsertSql(stmt)
         return self
+    
+    def attach_as_datastream(self: StatementSet):
+        """
+        combine table api statements into datastream
```

Comment:
Could we reference the documentation from the Java attachAsDataStream?

Reply:
I'll do that and adapt the docs to the python

Reply:
@billyrrr Thanks. Besides, there are test failures(check style issues): https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=36800&view=logs&j=9cada3cb-c1d3-5621-16da-0f718fb86602&t=c67e71ed-6451-5d26-8920-5a8cf9651901. Please also take a look when updating the PR~